index.css

```diff
@@ -206,23 +206,33 @@ hr {
 
 /***MODAL***/
 /**SIGN UP**/
+
+.modal-wrapper{
+    position: fixed;
+    top: 0;
+    bottom:0;
+    left: 0;
+    right: 0;
+    width: 100vw;
+    height: 100vh;
```

Comment:
typically the height should be set to a min-height so if you have extra content it doesn't cut off